drivers/hmis/app/models/hmis/hud/income_benefit.rb

```diff
@@ -11,6 +11,8 @@ class Hmis::Hud::IncomeBenefit < Hmis::Hud::Base
   self.sequence_name = "public.\"#{table_name}_id_seq\""
   include ::HmisStructure::IncomeBenefit
   include ::Hmis::Hud::Concerns::Shared
+  # FIXME: #7576
+  include ::Hmis::Hud::Concerns::WithStrictAttributes
```

Comment:
preserved this to keep the tests passing, income benefits are probably most benefited by this validation